docs/product-user-guide/src/main/asciidoc/chap-red-hat-jboss-dashboard-builder.adoc

```diff
@@ -1,83 +1,173 @@
 [id='_chap_red_hat_jboss_dashboard_builder']
 = Red Hat JBoss Dashboard Builder
 
-Red Hat JBoss Dashboard Builder
- is a web-based dashboard application that provides Business Activity Monitoring (BAM) support, that is, visualization tools for monitored metrics (Key Performance Indicators, or KPIs) in real time.
-It comes integrated in the Business Central environment under the *Dashboards* menu.
+Red Hat JBoss Dashboard Builder application supports Business Activity Monitoring (BAM) using visualization tools for monitoring metrics, such as Key Performance Indicators and process statuses. With the Dashboard Builder, you can compose custom business dashboards by mixing data coming from various data sources and displaying them as graphs or tables, as well as creating custom title navigation.
 
-It comes with a dashboard that requests information from the Red Hat JBoss BPM Suite Execution Engine and provides real-time information on its runtime data; however, you can also create custom dashboards over other data sources, which leaves the application relatively standalone.
+Dashboard Builder is connected to the local JBoss BPM Suite engine by default, and acquires the data for its JBoss BPM Suite Dashboard indicators. Data sources contain custom data that can be used to create data sets. Dashboard Builder can then use these data sets to create custom perspectives. The following types of data sets are supported:
 
-[float]
-== What is Business Activity Monitoring?
+Databases::
+This uses an SQL query for obtaining data from an external database.
```

Comment:
should be "This uses an SQL query *or Table name* for obtaining..."

Reply:
ack, done.